doc/nrf/doc_styleguide.rst

```diff
@@ -25,6 +21,7 @@ The |NCS| documentation follows the Zephyr style guide, and adds a few more rest
 * Do not use consecutive headings without intervening text.
 * For readability reasons, start every sentence on a new line in the source files.
   In the output, consecutive lines without blank lines in between are combined into one paragraph.
+* For the conceptual documentation written in RST, you can have more than 100 characters per line. The requirement for 100 characters per line applies only to the code documentation written in Doxygen.
```

Comment:
I think the limit is 80.

Reply:
Line break before the second sentence. ;)

Reply:
I would be a bit stronger here and state clearly that one sentence should be in one line, without line break - and not only that this is allowed.
The reason for this is that it makes commenting on sentences so much easier (because you will usually suggest a change to a sentence, not to the ten words on line 2 of the sentence).

Reply:
Not sure I understand where to make an edit based on your last comment.

Reply:
* For readability reasons, start every sentence on a new line in the source files and do not add line breaks within the sentence.
  In the output, consecutive lines without blank lines in between are combined into one paragraph.
  Note that the requirement of a maximum of 80 characters per line does not apply to RST files.